Sources/SwiftSyntax/gyb_generated/Trivia.swift

```diff
@@ -260,6 +260,15 @@ public struct Trivia {
   }
 }
 
+extension Trivia: CustomDebugStringConvertible {
+   public var debugDescription: String {
+     if count == 1, let first {
+       return first.debugDescription
+     }
+     return "[" + map({ $0.debugDescription }).joined(separator: ", ") + "]"
```

Comment:
```suggestion
     return "[" + map(\.debugDescription).joined(separator: ", ") + "]"
```

Reply:
Gyb-generated so will be fixed by the above one 🙇

Reply:
Ah, I missed that this was a generated file 😄